code/game/objects/items/weapons/explosives.dm

```diff
@@ -90,3 +98,50 @@
 
 /obj/item/plastique/attack(mob/M as mob, mob/user as mob, def_zone)
 	return
+
+/obj/item/plastique/cyborg
+	name = "plastic explosives dispenser"
+	desc = "A stationbound-mounted C4 dispenser, how thrilling!"
+	desc_antag = "When used, this dispenser will deploy C4 on a target, upon which it will enter a charging state. After two minutes, it will restock a new C4 bundle."
+	var/can_deploy = TRUE
+	var/recharge_time = 2 MINUTES
```

Comment:
That recharge time is way too small.
Unlimited C4 with a 2min recharge time is a nogo.